crates/hir-ty/src/diagnostics/match_check/pat_analysis.rs

```diff
@@ -460,7 +460,9 @@ impl<'p> TypeCx for MatchCheckCtx<'p> {
         _f: &mut fmt::Formatter<'_>,
         _pat: &rustc_pattern_analysis::pat::DeconstructedPat<'_, Self>,
     ) -> fmt::Result {
-        unimplemented!()
+        // TODO: implement this
```

Comment:
```suggestion
        // FIXME: implement this
```

CI fails on TODO comments

Reply:
done!